sdk-core/src/main/java/com/reach5/identity/sdk/core/ReachFive.kt

```diff
@@ -196,15 +197,22 @@ class ReachFive(val activity: Activity, val sdkConfig: SdkConfig, val providersC
     }
 
     fun updatePassword(
-        authToken: AuthToken,
-        updatePhoneNumberRequest: UpdatePasswordRequest,
+        updatePasswordRequest: UpdatePasswordRequest,
         successWithNoContent: SuccessWithNoContent<Unit>,
         failure: Failure<ReachFiveError>
     ) {
+        val authToken = getAccessToken(updatePasswordRequest)
+
+        val headers = if (authToken !== null) {
```

Comment:
```suggestion
        val headers = if (authToken != null) {
```

Reply:
Prefer `= authToken?.let { mapOf("Authorization" to formatAuthorization(it)) } ?: emptyMap()` instead of if/else